device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-C64/qos.json.j2

```diff
@@ -277,4 +277,24 @@
 
 {% endif %}
 
+{%- macro generate_wred_profiles() %}
+    "WRED_PROFILE": {
+        "AZURE_LOSSLESS" : {
+            "wred_green_enable"      : "true",
+            "wred_yellow_enable"     : "true",
+            "wred_red_enable"        : "true",
+            "ecn"                    : "ecn_all",
+            "green_max_threshold"    : "10000000",
+            "green_min_threshold"    : "2000000",
+            "yellow_max_threshold"   : "10000000",
+            "yellow_min_threshold"   : "2000000",
+            "red_max_threshold"      : "10000000",
+            "red_min_threshold"      : "2000000",
+            "green_drop_probability" : "5",
+            "yellow_drop_probability": "5",
+            "red_drop_probability"   : "5"
+        }
+    },
+{%- endmacro %}
+
```

Comment:
Suggest changing other HWSKUs as well if there is no side effect.
- Mellanox-SN4600C-D100C12S2
- Mellanox-SN4600C-D112C8 
- Mellanox-SN4600C-D48C40

Reply:
Currently Mellanox-SN4600C-D48C40/qos.json.j2 is a softlink to Mellanox-SN4600C-C64/qos.json.j2. So it should work.

For other 2 SKUs, it is a softlink to ACS-MSN4600C, which might need other values. We can update it later.